subprojects/inline/inline.gradle

```diff
@@ -12,3 +12,7 @@ tasks.javadoc.enabled = false
 //required by the "StressTest.java" and "OneLinerStubStressTest.java"
 test.maxHeapSize = "256m"
 retryTest.maxHeapSize = "256m"
+
+test {
+    jvmArgs '--illegal-access=debug'
```

Comment:
This means that the build will now fail when an Illegal Access is performed? Nice!

Reply:
It will log it in more detail but fail would be the much more sensible option. Weird I did not think of it, that's of course a much better value...